[2024-09-25T11:09:29.747+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-25T11:09:29.784+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.pipeline.warehouse.step_1.load.customers manual__2024-09-25T11:05:57.968197+00:00 [queued]>
[2024-09-25T11:09:29.810+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.pipeline.warehouse.step_1.load.customers manual__2024-09-25T11:05:57.968197+00:00 [queued]>
[2024-09-25T11:09:29.811+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-25T11:09:29.836+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): pipeline.warehouse.step_1.load.customers> on 2024-09-25 11:05:57.968197+00:00
[2024-09-25T11:09:29.849+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=734) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-25T11:09:29.853+0000] {standard_task_runner.py:72} INFO - Started process 736 to run task
[2024-09-25T11:09:29.854+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'pipeline.warehouse.step_1.load.customers', 'manual__2024-09-25T11:05:57.968197+00:00', '--job-id', '188', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline/run.py', '--cfg-path', '/tmp/tmpnsvrgny1']
[2024-09-25T11:09:29.861+0000] {standard_task_runner.py:105} INFO - Job 188: Subtask pipeline.warehouse.step_1.load.customers
[2024-09-25T11:09:29.940+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.pipeline.warehouse.step_1.load.customers manual__2024-09-25T11:05:57.968197+00:00 [running]> on host e10075799413
[2024-09-25T11:09:30.123+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='pipeline.warehouse.step_1.load.customers' AIRFLOW_CTX_EXECUTION_DATE='2024-09-25T11:05:57.968197+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-25T11:05:57.968197+00:00'
[2024-09-25T11:09:30.127+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-25T11:09:30.151+0000] {base.py:84} INFO - Retrieving connection 'minio'
[2024-09-25T11:09:30.349+0000] {base.py:84} INFO - Retrieving connection 'warehouse_db'
[2024-09-25T11:09:52.535+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-25T11:09:52.557+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-25T11:09:52.558+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=pipeline.warehouse.step_1.load.customers, run_id=manual__2024-09-25T11:05:57.968197+00:00, execution_date=20240925T110557, start_date=20240925T110929, end_date=20240925T110952
[2024-09-25T11:09:52.677+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-25T11:09:52.769+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-25T11:09:52.774+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
